patterns/2-structured/service-vs-library.md

```diff
@@ -42,20 +42,24 @@ do not affect their own downstream customers.
 Severity levels for the same types of errors may be different across team
 boundaries due to different SLA definitions per team/customer relationship.
 
+Teams may have different security or regulatory constraints governing their deployments.
+
 ## Solutions
 
 Decouple responsibility for source code from deployment: Both teams work to
 identify exactly where there is overlap and synergies.
 
 Only shared source code is kept as part of the InnerSource project with shared
-responsibility.
+responsibility. Test and release packaging should be included in the shared source to allow shared change validation.
```

Comment:
I liked this summary in the PR description:
> Our main learning point in this space is to ensure the "shared code" is coherent in that it also includes all testing code (incl integration tests if available) and keep as much of the CI pipeline shared as is possible.

Is this what you are describin in your changes in L53 and L61/62?

Reply:
Yes -- although the L61/62 are more a side comment that it's handy when contributors can see/copy the various deployments. I've combined the comment you quoted into the pattern text here as I agree it's a point worth expanding so it's understood.